bioimageio/core/proc_ops.py

```diff
@@ -688,3 +692,113 @@ def get_proc_class(proc_spec: ProcDescr):
         return ZeroMeanUnitVariance
     else:
         assert_never(proc_spec)
+
+
+def preproc_v4_to_processing(
+    inp: v0_4.InputTensorDescr,
+    proc_spec: v0_4.PreprocessingDescr,
+) -> Processing:
+    member_id = TensorId(str(inp.name))
+    if isinstance(proc_spec, v0_4.BinarizeDescr):
+        return Binarize.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ClipDescr):
+        return Clip.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ScaleLinearDescr):
+        return ScaleLinear.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ScaleRangeDescr):
+        return ScaleRange.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.SigmoidDescr):
+        return Sigmoid.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ZeroMeanUnitVarianceDescr):
+        if proc_spec.kwargs.mode == "fixed":
+            axes = inp.axes
+            v5_proc_spec = _convert_proc(proc_spec, axes)
+            assert isinstance(
+                v5_proc_spec, v0_5.FixedZeroMeanUnitVarianceDescr
+            )  # FIXME
+            return FixedZeroMeanUnitVariance.from_proc_descr(v5_proc_spec, member_id)
+        else:
+            return ZeroMeanUnitVariance.from_proc_descr(proc_spec, member_id)
+    else:
+        assert_never(proc_spec)
+
+
+def postproc_v4_to_processing(
+    inp: v0_4.OutputTensorDescr,
+    proc_spec: v0_4.PostprocessingDescr,
+) -> Processing:
+    member_id = TensorId(str(inp.name))
+    if isinstance(proc_spec, v0_4.BinarizeDescr):
+        return Binarize.from_proc_descr(proc_spec, member_id)
+    if isinstance(proc_spec, v0_4.ScaleMeanVarianceDescr):
+        return ScaleMeanVariance.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ClipDescr):
+        return Clip.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ScaleLinearDescr):
+        return ScaleLinear.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ScaleRangeDescr):
+        return ScaleRange.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.SigmoidDescr):
+        return Sigmoid.from_proc_descr(proc_spec, member_id)
+    elif isinstance(proc_spec, v0_4.ZeroMeanUnitVarianceDescr):
+        if proc_spec.kwargs.mode == "fixed":
+            axes = inp.axes
+            v5_proc_spec = _convert_proc(proc_spec, axes)
+            assert isinstance(
+                v5_proc_spec, v0_5.FixedZeroMeanUnitVarianceDescr
+            )  # FIXME
+            return FixedZeroMeanUnitVariance.from_proc_descr(v5_proc_spec, member_id)
+        else:
+            return ZeroMeanUnitVariance.from_proc_descr(proc_spec, member_id)
+    else:
+        assert_never(proc_spec)
+
+
+def preproc_v5_to_processing(
```

Comment:
Might be a bit pedantic..., but couldn't we just merge `preproc_v5... ` and `postproc_v5...` and annotate with the union?
Those two have a lot of code duplication and pre and postproc descr are just different unions of the same set of descriptions.

Same for v4.

Reply:
Fair point. I guess I was thinking that there would be both `Preprocessing` and `Postprocessing` classes, instead of just `Processing`. I unified them now